kolibri/plugins/management/assets/src/device_management/views/manage-content-page/channels-grid.vue

```diff
@@ -2,11 +2,11 @@
 
   <div>
     <transition mode="out-in">
-      <p class="core-text-alert" v-if="sortedChannels.length===0 && !this.channelsLoading">
+      <p class="core-text-alert" v-if="sortedChannels.length===0 && !channelsLoading">
```

Comment:
Probably need to manually enforce ESLint rules like 'operator spacing' for things like `sortedChannels.length===0` when they appear in directives.